pkg/regressionallowances/consecutiveoverrides/overrides.json

```diff
@@ -1 +1,51 @@
-{}
+{    
+   "JiraComponent": "kube-apiserver",
```

Comment:
This file looks like you want it a directory above in regressions_4.18.json. 

Only the budget burn needs to be in this file, and the syntax is simpler, see the README in this dir.